problem_builder/tests/integration/test_mentoring.py

```diff
@@ -334,6 +338,7 @@ def _test_persistence(self, options, after_reload_checks):
 
             # Now reload the page...
             mentoring = self.reload_student_view()
+            time.sleep(3)
```

Comment:
This is not an acceptable solution, please use a selenium wait for the UI to update.

Reply:
Addressed